hack/jenkins/test-dockerized.sh

```diff
@@ -47,14 +36,10 @@ pushd "./hack/tools" >/dev/null
   GO111MODULE=on go install gotest.tools/gotestsum
 popd >/dev/null
 
-# Enable the Go race detector.
-export KUBE_RACE=-race
```

Comment:
we set this elsewhere. this is redundant / useless / another way to make this more confusing.